docker/setup_postgres.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-DB_NAME=${1:-$DB_USERNAME}  # Until we move off template deploy and onto Kubernetes with the new DB
+DB_NAME=${DB_NAME:-$DB_USERNAME}  # Until we move off template deploy and onto Kubernetes with the new DB
```

Comment:
Feels like a related change should be made in `<root>/setup_postgres.sh` too.

Reply:
Yeah that was a complete oversight, I hadn't even noticed the existence of that file when I was doing the env vars work.
👍to separate PR for that, although I'd be in favour of leaving this bit in here now